### PR TITLE
Add several more instructions to release lead guide, particularly inf…

### PR DESCRIPTION
**Pull request by jberkus**
…ormation about choosing a team, required time commitments, and some updates on process and requirements since the guide was first written.  Not finished yet, but still helpful updates.

### CODE REVIEW

File: release-team/role-handbooks/release-lead.md [hunk 1]
```diff
@@ -34,6 +34,33 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
...
+* Community meetings once a week.
+
+## Choosing a Release Team
+
+One of your first, and definitely your most important, duties as Release Lead is to select a Release Team.  Currently there are 10 roles to fill on the Release Team, plus one or more shadows (trainees) of each of those roles. each of which has specific requirements. Aside from any specifics, in general you are looking for candidates who:
+
+1. Can commit to the amount of time required across the release cycle;
+2. Are enthusiastic about being on the release team;
+3. Are members of the relevant SIGs for their role;
+4. And have some prior experience with contributing to Kubernetes.
```
[the remark this answers is not in the document]
Comment (jberkus) [reply release-team/role-handbooks/release-lead.md]: shadowing is prior experience with contributing, but I can spell that out.

**Comment by jberkus** [comment]
Updated per feedback.

File: release-team/role-handbooks/release-lead.md [hunk 2]
```diff
@@ -50,12 +50,12 @@ Among the specific time commitments you have are:
...
 
-One of your first, and definitely your most important, duties as Release Lead is to select a Release Team.  Currently there are 10 roles to fill on the Release Team, plus one or more shadows (trainees) of each of those roles. each of which has specific requirements. Aside from any specifics, in general you are looking for candidates who:
+One of your first, and definitely your most important, duties as Release Lead is to select a Release Team in consultation with SIG-Release.  Currently there are 10 roles to fill on the Release Team, plus one or more shadows (trainees) of each of those roles. each of which has specific requirements. Aside from any specifics, in general you are looking for candidates who:
 
 1. Can commit to the amount of time required across the release cycle;
 2. Are enthusiastic about being on the release team;
-3. Are members of the relevant SIGs for their role;
-4. And have some prior experience with contributing to Kubernetes.
+3. Are members of the relevant SIGs for their role, if applicable;
+4. And have some prior experience with contributing to Kubernetes (such as shadowing a role for a prior release).
```
Comment (tpepper) [inline release-team/role-handbooks/release-lead.md]: I was thinking something like, eg:

4. Lead roles especially need some prior experience with contributing to Kubernetes (such as shadowing a role for the prior release), but shadowing may represent an entry point or growth opportunity.

Comment (tpepper) [reply release-team/role-handbooks/release-lead.md]: As is the text seems sufficient.  I'm fine with it either way really, just saying out loud what I thought I heard Caleb and you saying, but didn't end up seeing in the .md text.

**Comment by tpepper** [comment]
@jberkus can you rebase

Poke @calebamiles ...are you ok with this now?

**Comment by jberkus** [comment]
@tpepper not until @calebamiles approves, no point in rebasing multiple times

File: release-team/role-handbooks/release-lead.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview:
 
-The release team leader role is responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring a retrospective happens.  The lead is also responsible for ensuring a successor is selected once the release is complete.  
+The release team leader role is responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring a retrospective happens.  The lead is also responsible for ensuring a successor is trained for future release cycles.
```
Comment (jdumars) [inline release-team/role-handbooks/release-lead.md]: The selection process is explicit here and not mutually exclusive to training someone else. The intent is for there to be someone committed to the next cycle, and the current release lead is responsible for ensuring that. I would say selected and trained.

File: release-team/role-handbooks/release-lead.md [hunk 4]
```diff
@@ -34,6 +34,33 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
...
+
+Among the specific time commitments you have are:
+
+* SIG-Release and Release Team meetings once a week during weeks 1-7.
+* Burndown meetings three to five times a week during weeks 8-12.
+* Community meetings once a week.
+
+## Choosing a Release Team
+
+One of your first, and definitely your most important, duties as Release Lead is to select a Release Team in consultation with SIG-Release.  Currently there are 10 roles to fill on the Release Team, plus one or more shadows (trainees) of each of those roles. each of which has specific requirements. Aside from any specifics, in general you are looking for candidates who:
```
Comment (jdumars) [inline release-team/role-handbooks/release-lead.md]: It's actually the responsibility of each role holder to try and recruit/train their replacement, or offer to perform the role again. The lead is only responsible for ensuring a full team is in place by the last responsible moment. In some cases, the lead may hunt for people, but it is not their responsibility until it cannot be filled.

File: release-team/role-handbooks/release-lead.md [hunk 5]
```diff
@@ -34,6 +34,33 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
...
+
+1. Can commit to the amount of time required across the release cycle;
+2. Are enthusiastic about being on the release team;
+3. Are members of the relevant SIGs for their role, if applicable;
+4. And have some prior experience with contributing to Kubernetes (such as shadowing a role for a prior release).
+
+In most cases, you will have a pool of candidates in the form of people who have previously shadowed the role in question, or have filled a different role on the Release Team and want to try something else.  Where possible, do select new team members over someone who has done the role several times, as we are trying to share experience across the organization.
+
+Also give some thought to ensuring diversity on the team, including members of more than one gender, ethnicity, nationality, and company affiliation.  This should not be hard to do, as prior release teams have had several women, several non-white folks, and staff from five or more companies.  In some cases, though, you may need to reach out to individuals you know and suggest that they apply for the team.
```
Comment (jdumars) [inline release-team/role-handbooks/release-lead.md]: I would use the term "underrepresented" in this context. Thank you for adding this.

File: release-team/role-handbooks/release-lead.md [hunk 6]
```diff
@@ -76,13 +103,13 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
...
 * The release lead is responsible for updating the [burndown template](https://docs.google.com/document/d/1zLnmDDOp_ko9Yh5uPJtgqPFD7GKq76fQsKaenXoMHzM/edit) ahead of the release (changing the milestone in links and anything else requested during the retrospective)
 
-* Since 1.8, releases have had unofficial food-based code names. There is no particular reason for this other than to have fun.  As release lead you get to pick your food.
-
-    * 1.8 - "Burrito"
+* Since 1.8, releases have had unofficial code names, chosen right before the release. There is no particular reason for this other than to have fun, and possibly provide a theme for release team gifts/shirts.
 
     * 1.9 - "Pumpkin"
 
-    * 1.10 - "Kiwi"
```
Comment (jdumars) [inline release-team/role-handbooks/release-lead.md]: Nice catch! I accidentally jumped the shark!

**Comment by justaugustus** [comment]
@tpepper @jberkus -- I've created a PR for RT selection criteria (https://github.com/kubernetes/sig-release/pull/271), which represents essentially what we did in 1.12, shifting RT selection to be a function of the Release Team as opposed to just RT Lead.

It's in the RT readme, as well, making it more discoverable.
PTAL when you have a chance.

**Comment by jberkus** [comment]
I won't have time to deal with completely re-building this PR until October, so if someone else wants to take the useful content from it, be my guest.

**Comment by justaugustus** [comment]
@jberkus -- I'll pick this up as it's related to #271.
/assign

**Pull request closed without merging**